src/helpers/glsl/Particles.vert.glsl

```diff
@@ -0,0 +1,11 @@
+uniform float time;
+uniform float radius;
+uniform float size;
+
+void main() {
+  vec3 n = curlNoise(position + time * 0.005);
+  vec3 p = n * radius;
+
+  gl_PointSize = size * 10.;
```

Comment:
maybe have size as a buffer attribute instead so that you can vary particle size easily?